### ISSUE TITLE
appendContentOnMatchOnly is not a valid input

### ISSUE THREAD

**Issue by angelod1as** [nefrob/pr-description#7]
## What's happening

I'm following the docs and did:

```
name: "Update PR description"

on:
  pull_request:

jobs:
  update-pr-description:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v3
      - name: Do action
        uses: nefrob/pr-description@v1.1.0
        with:
          content: ${{github.event.number}}
          regex: "PR_NUMBER"
          regexFlags: i
          token: ${{ secrets.GITHUB_TOKEN }}
          appendContentOnMatchOnly: true      <---- here
```

which resulted in:

![image](https://github.com/nefrob/pr-description/assets/13950513/ccba6c51-cafd-439c-b609-1b7a75affcd9)

Also, my content is appended to the description even with the property `true`

## Expected:

The property should work.

**Comment by bads77** [comment]
Plus one:

`Warning: Unexpected input(s) 'appendContentOnMatchOnly', valid inputs are ['content', 'contentIsFilePath', 'regex', 'regexFlags', 'token']`

This flag is not working

**Comment by nefrob** [comment]
@angelod1as I just released this new configurable option in [v1.1.1](https://github.com/nefrob/pr-description/releases/tag/v1.1.1) earlier today. Can you try bumping the version from your above example as that might be the issue?

@bads77 what version of the action were you using?

**Comment by bads77** [comment]
> [1.1.1](https://github.com/nefrob/pr-description/releases/tag/v1.1.1)

Hi @nefrob, I was using 1.1.0 as well.

**Comment by angelod1as** [comment]
@nefrob Apparently it's fixed. Gimme a few more hours to test with other PRs and I'll return here and close if it's ok.

**Comment by angelod1as** [comment]
Fixed in [1.1.1](https://github.com/nefrob/pr-description/releases/tag/v1.1.1)

**Comment by nefrob** [comment]
Great, thank you for checking! I'll make sure the readme and github marketplace description are up to date as well.
